src/Executables/Benchmark/Benchmark.cpp

```diff
@@ -103,6 +104,7 @@ void bench_all_gradient(benchmark::State& state) {  // NOLINT
 BENCHMARK(bench_all_gradient);
 }  // namespace
 
-BENCHMARK_MAIN()
-
-#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.tpp"
+// Some compilers require a semi-colon here with some versions of Benchmark. If
+// you get "xxpected initialzer at end of input" errors then you need to add the
+// semi-colon.
+BENCHMARK_MAIN()  // ;
```

Comment:
This change seems unrelated, and "expected" and "initializer" are spelled wrong.